.pre-commit-config.yaml

```diff
@@ -0,0 +1,38 @@
+---
+exclude: .crt$
+repos:
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.44.0
+    hooks:
+      - id: markdownlint
+        args:
+          - --dot
+          - --fix
+  - repo: https://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.5.5
+    hooks:
+      - id: forbid-tabs
+  - repo: https://github.com/rbubley/mirrors-prettier
+    rev: v3.5.3
+    hooks:
+      - id: prettier
```

Comment:
What does twine output? Does `prettier` make sense?

Reply:
prettier worked like a charm

Reply:
Actually it might have just added a new line